getting-started/client-blazor.md

```diff
@@ -1,14 +1,14 @@
 ---
 title: Blazor WebAssembly (Tutorial)
-page_title: First Steps with Blazor WebAssembly Apps and Telerik UI for Blazor
-description: Make your first steps with Telerik UI for Blazor and build an app that hosts the Blazor UI components client-side (by using Blazor WebAssembly, WASM).
+page_title: First Steps with Blazor WebAssembly Apps - Telerik UI for Blazor
```

Comment:
The ` - Telerik UI for Blazor` gets added automatically during build, so you don't need to have it in your source. So right now you have this part appear twice in the meta data. Fix as needed throughout the PR.

![image](https://github.com/telerik/blazor-docs/assets/3539221/d2132e45-6c2b-4921-82ff-8769580ce0ea)

```suggestion
page_title: First Steps with Blazor WebAssembly Apps
```